hz: 20
turbo:
  steps: 4
  linear_forward_min: 0.5
  linear_forward_max: 2.0
  linear_backward_min: 0.25
  linear_backward_max: 0.5
  angular_min: 0.1
  angular_max: 0.7
